### ISSUE TITLE
sql: tuple serialization error with aggregation

### ISSUE THREAD

**Issue by RaduBerinde** [cockroachdb/cockroach#28382]
```
root@127.233.3.32:43015/defaultdb> select count(distinct t.*) from (values (1), (2), (1)) AS t(x);
pq: syntax error at or near "as"
DETAIL: source SQL:
SET ROW (((@1) AS x))
               ^
HINT: try \h SET SESSION
```

**Comment by RaduBerinde** [comment]
CC @jordanlewis @arjunravinarayan

**Comment by jordanlewis** [comment]
This is fixed on my branch, though I don't remember why... in any case, it'll be fixed soon.

**Comment by arjunravinarayan** [comment]
@jordanlewis can this be closed now that #27863 is merged?

**Comment by jordanlewis** [comment]
Yeah, thanks!
